### PR TITLE
Add a form for publishers to submit new papers

### PR DESCRIPTION
**Pull request by acabunoc**
* input for DOI & author emails
* partially addresses #22

### CODE REVIEW

**Comment by acabunoc** [comment]
@alicoding -- I added a react page. Can you r?

I don't know a lot about react forms.

File: templates/pages/paper.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+var React = require('react'),
+    fetch = require('isomorphic-fetch'),
+    Url = require('url'),
+    Page = require('../components/page.jsx');
+
+var Issue = React.createClass({
+  email_id: 0,
```
Comment (alicoding) [inline templates/pages/paper.jsx]: Why don't you include this in the `state` object instead?

File: templates/pages/paper.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  render: function() {
+    return (
+      <Page>
+        <h1>Submit a Paper</h1>
+        <p>This is a simple prototype demonstrating using a form to submit papers to Paper Badger, our contributorship badges prototype. In future versions we will integrate with publisher submission pipelines.</p>
+        <form className="pure-form pure-form-aligned" onSubmit={this.handleSubmit}>
+            <fieldset>
+                <div className="pure-control-group">
+                    <label for="doi">DOI</label>
+                    <input ref="doi" id="doi" type="text" placeholder="Paper DOI" />
```
Comment (alicoding) [inline templates/pages/paper.jsx]: Maybe you want to `linkState` this field using `valueLink` with your `state` object and that way you can update the state as you go and validate them easily here. https://facebook.github.io/react/docs/two-way-binding-helpers.html

**Comment by alicoding** [comment]
Just a bit more work using two way binding as suggested in the URL provided in the comment above.

**Comment by acabunoc** [comment]
wow, linkstate is way nicer! Thanks @alicoding

**Comment by acabunoc** [comment]
@alicoding -- I added linkstate a couple places, but I had some trouble with the dynamic inputs (you can add/remove emails, see below)

submit-paper

**Comment by acabunoc** [comment]
@alicoding - I'm going to replace the dynamic inputs with one textarea. Publishers can add an email on each line instead.

https://github.com/mozillascience/PaperBadger/issues/22#issuecomment-123715199

It'll be easier with linkstate, too

**Comment by acabunoc** [comment]
@alicoding,

Simplified to use just a textarea! Added a new issue for validation (#76). Otherwise, good for review!

File: templates/pages/paper.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+      );
+  }
+});
+
+
+module.exports = Issue;
+
+
+
```
Comment (alicoding) [inline templates/pages/paper.jsx]: just remove this extra lines :P

**Comment by alicoding** [comment]
r+ with just one nit :P

**Comment by acabunoc** [comment]
done, thanks @alicoding! You da best

**Pull request merged**